db/txn_db.go

```diff
@@ -213,7 +220,12 @@ func (db *implicitTxnDB) AddSchema(ctx context.Context, schemaString string) ([]
 // All schema types provided must not exist prior to calling this, and they may not reference existing
 // types previously defined.
 func (db *explicitTxnDB) AddSchema(ctx context.Context, schemaString string) ([]client.CollectionDescription, error) {
-	return db.addSchema(ctx, db.txn, schemaString)
+	cols, err := db.addSchema(ctx, db.txn, schemaString)
+	if err != nil {
+		log.ErrorContextE(ctx, errCouldNotAddSchema, err)
```

Comment:
todo: Please remove these log calls.  Even if you really really want to keep them, this file is not the place to log errors in.

Reply:
This was added because I noticed no error pops up / logs when a schema adding fails.

I don't care much to do this in this PR if you like I can drop ``` PR(-): Log Server Error On 'Schema Adding Failure' ``` entirely. LMK

Reply:
Yeah please drop that commit, something in http/cli must be hiding the error if you are not seeing any - I see them using the Go client.

I'll open a ticket to investigate but please remove this logging.

Reply:
https://github.com/sourcenetwork/defradb/issues/2447

Reply:
- [ ] Drop ``` PR(-): Log Server Error On 'Schema Adding Failure' ```